### ISSUE TITLE
error in calculating curvature?

### ISSUE THREAD

**Issue by JinlongLi2016** [fjp/frenet#9]
https://github.com/fjp/frenet/blob/a7e8dd5c2252cfec7f22e3cd61ec8bc282528049/python/src/cubic_spline_planner.py#L167

It seems that the divisor should be  $(x'**2 + y'**2)^{3/2}$?

**Comment by fjp** [comment]
You are correct, thanks for spotting this error. Feel free to open a PR to fix this mistake.
